udata_front/theme/gouvfr/assets/js/components/DataservicesSearch/DataservicesSearch.vue

```diff
@@ -0,0 +1,213 @@
+<template>
+  <form class="fr-pt-3v" @submit.prevent="search">
+    <div class="fr-grid-row fr-grid-row--middle justify-between" ref="searchRef" data-cy="search">
+      <section class="fr-search-bar fr-search-bar--lg w-100">
+        <label class="fr-label" :for="searchId">
+          {{ t("Search") }}
+        </label>
+        <input
+          :id="searchId"
+          type="search"
+          v-model="searchQuery"
+          ref="searchInput"
+          class="fr-input"
+          :aria-label="t('Search...')"
+          :placeholder="t('Search...')"
+          data-cy="search-input"
+          data-testid="search-input"
+        />
+        <button class="fr-btn" type="submit">
+          {{ t('Search') }}
+        </button>
+      </section>
+    </div>
+    <div class="fr-grid-row fr-mt-1w fr-mt-md-5v">
+      <div class="fr-col-12 fr-col-md-4 fr-col-lg-3">
+        <nav class="fr-sidemenu" aria-labelledby="fr-sidemenu-title">
+          <div class="fr-sidemenu__inner">
+            <button
+              class="fr-sidemenu__btn fr-mt-1w"
+              hidden
+              aria-controls="fr-sidemenu-wrapper"
+              aria-expanded="false"
+            >
+              {{ t('Filter results') }}
+            </button>
+            <div class="fr-collapse" id="fr-sidemenu-wrapper">
+              <div class="fr-sidemenu__title fr-mb-3v" id="fr-sidemenu-title">{{ t('Filters') }}</div>
+              <div class="fr-grid-row fr-grid-row--gutters">
+                <div class="fr-col-12">
+                  <MultiSelect
+                    :placeholder="t('Organizations')"
+                    :searchPlaceholder="t('Search an organization...')"
+                    :allOption="t('All organizations')"
+                    listUrl="/organizations/?sort=-followers"
+                    suggestUrl="/organizations/suggest/"
+                    entityUrl="/organizations/"
+                    :values="organization"
+                    @change="(value: string) => organization = value"
+                    :isBlue="true"
+                  />
+                </div>
+                <div class="fr-col-12">
+                  <div class="fr-select-group">
+                    <label class="fr-label" :for="isRestrictedId">
+                      {{ t("Access terms") }}
+                    </label>
+                    <select class="fr-select" :id="isRestrictedId" v-model="isRestricted">
```

Comment:
Changes to this filter should all reset the page to 1

Reply:
Done in 92e2b8ac